config/config.go

```diff
@@ -1,35 +1,66 @@
 package config
 
 import (
-	"fmt"
 	"log"
-	"path/filepath"
-
-	"github.com/spf13/viper"
+	"os"
 )
 
-// Init is an exported method that takes the environment starts the viper
-// (external lib) and returns the configuration struct.
-func Init(cfgFile string) {
-	if cfgFile != "" { // enable ability to specify config file via flag
-		viper.SetConfigFile(cfgFile)
-	} else {
-		viper.SetConfigName(".taco") // name of config file (without extension)
-		viper.AddConfigPath("$HOME") // adding home directory as first search path
+type Config struct {
+	AWS_Dynamo_Region string
+	Dynamo_Db         string
+	Disable_SSL       bool
+	Table_Name        string
+}
+
+func NewConfig() *Config {
+	return &Config{
+		AWS_Dynamo_Region: aws_dynamo_region(),
+		Dynamo_Db:         dynamo_db(),
+		Disable_SSL:       disable_ssl(),
+		Table_Name:        table_name(),
+	}
+}
+
+func aws_dynamo_region() string {
+	var region string
+	region = os.Getenv("AWS_DYNAMO_REGION")
+	if region == "" {
+		region = "localstack"
+		log.Printf("AWS_DYNAMO_REGION: Using default [localstack].")
+	}
+	log.Printf("AWS_DYNAMO_REGION: Found setting [%s]", region)
+	return region
+}
+
+func dynamo_db() string {
+	var db string
+	db = os.Getenv("DYNAMO_DB")
+	if db == "" {
+		db = "localhost:4569"
+		log.Printf("DB_ENDPOINT: Using default [localhost:4569].")
 	}
-	viper.AutomaticEnv() // read in environment variables that match
-	// If a config file is found, read it in.
-	err := viper.ReadInConfig()
-	if err == nil {
-		fmt.Println("Using config file:", viper.ConfigFileUsed())
+	log.Printf("DB_ENDPOINT: Found setting [%s]", db)
+	return db
+}
+
+func disable_ssl() bool {
+	var disablessl string
+	disablessl = os.Getenv("DISABLE_SSL")
```

Comment:
while I don't think we are ever going to set this to `false` (we will be running dynamo within a VPC) it would be good to mark this as a dynamo env var. Something like: `DYNAMODB_DISABLE_SSL` ?